app/src/main/java/com/example/jasmiensofiecels/wordly/viewModel/DictionaryViewModel.java

```diff
@@ -0,0 +1,35 @@
+package com.example.jasmiensofiecels.wordly.viewModel;
+
+import android.arch.lifecycle.LiveData;
+import android.arch.lifecycle.MutableLiveData;
+import android.arch.lifecycle.ViewModel;
+
+import com.example.jasmiensofiecels.wordly.service.model.OxfordEntry.Example;
+import com.example.jasmiensofiecels.wordly.service.repository.DictionaryRepository;
+
+/**
+ * The purpose of this class is to represent the viewModel that gets observed by the view
+ * and calls the model (DictionaryRepository) to get data.
+ *
+ * Created by Jasmien Cels on 04/04/2018.
+ */
+
+public class DictionaryViewModel extends ViewModel {
+
+    public DictionaryRepository repository;
+
+    public static MutableLiveData<Example> resultObservable = new MutableLiveData<>();
+
+    public DictionaryViewModel(DictionaryRepository repository) {
+        this.repository = repository;
+    }
+
+    public void onWordRefresh(MutableLiveData<String> word) {
+        resultObservable = repository.getWordOfTheDay("en", word);
```

Comment:
I would prefer the use of [`postValue()`](https://developer.android.com/reference/android/arch/lifecycle/MutableLiveData.html#postValue(T)) for threading purposes. This line should read:

`resultsObservable.postValue(repository.getWordOfTheDay("en", word));`

At the moment you have a `repository` returning a `LiveData`. This goes against clean architecture as all repositories should be doing is fetching data and nothing more. It should have no knowledge of `LiveData`, just the data. But how would you also do an error response?